src/main/java/nl/utwente/presto/tezos/TezosRange.java

```diff
@@ -13,7 +13,7 @@ public static TezosRange fromMarkers(String column, Marker low, Marker high) {
         long start;
         long end;
         if (low.isLowerUnbounded()) {
-            start = 1L;
+            start = 0L;
```

Comment:
Tezos starts counting at 0 (i.e. the first block has height 0)

Reply:
Makes sense